mlflow/R/mlflow/man-roxygen/roxlate-client.R

```diff
@@ -1,3 +1,2 @@
 #' @param client (Optional) An `mlflow_client` object.
-#' @family tracking functions
```

Comment:
Removed this family annotation, which causes the @seealso blobs to disappear from the R public API docs. The main downside is that if users build R docs on their own / view docs for an individual function like `mlflow_log_metric` e.g. in RStudio, there won't be cross-links to other tracking API docs. However this seems like a worthwhile tradeoff for having clean public docs.